site/data/en-US/docs/custom-connect-button.mdx

```diff
@@ -19,6 +19,7 @@ export const YourApp = () => {
     <ConnectButton.Custom>
       {({
         account,
+        disconnecting,
```

Comment:
This technically won't be a breaking change. If we were to merge this the only thing that would impact is if `disconnecting` is true, then `openConnectModal` with just return a `noop` function. [Here](https://github.com/rainbow-me/rainbowkit/blob/main/packages/rainbowkit/src/components/ConnectButton/ConnectButtonRenderer.tsx#L154-L156) is an example.

Reply:
For injected wallets it should happen super fast, but for WalletConnect in some scenarios might take time. Of course depending on their socket server.